spec/unit/cds_synchronizer_spec.rb

```diff
@@ -0,0 +1,184 @@
+# rubocop:disable RSpec/MultipleExpectations
```

Comment:
We don't need this with the :aggregate_failures switch on the context/example with multiple expectations

Reply:
Generally we shouldn't have these file-specific overrides of rubocop rules

Reply:
I didn't find a way to do bellow test without multiple expectation in the same example.

```
context 'when pending CDS update does not respect the sequence' do
      let(:pending_date) { applied_date + 2.days }

      it 'raises wrong sequence error and notifies Slack app' do
        allow(SlackNotifierService).to receive(:call)

        expect { described_class.apply }.to raise_error(TariffSynchronizer::FailedUpdatesError)
        # allow(described_class).to receive(:apply).and_raise(TariffSynchronizer::FailedUpdatesError)

        described_class.apply

        expect(SlackNotifierService).to have_received(:call)
      end
    end
```

Tried with commented stub, but it doesn't work in this test, any few other similar tests in same spec. This might be the reason why ovverrides added originally.